datadog_checks_base/CHANGELOG.md

```diff
@@ -20,7 +20,7 @@
 
 * Added Postgres cross-org telemetry metrics. ([#18758](https://github.com/DataDog/integrations-core/pull/18758))
 
-## 37.0.0 / 2024-09-19
+## 37.0.0 / 2024-09-19 / Agent 7.58.0
```

Comment:
this doesn't seem right:

1. we shouldn't be adding 7.58 in this PR at all
2. supposedly  v36.16.0 was released with 7.58

Reply:
Fixed